src/main/kotlin/net/irisshaders/lilybot/extensions/config/Config.kt

```diff
@@ -36,99 +34,72 @@ class Config : Extension() {
 				name = "set"
 				description = "Set the config"
 
-				// We only want admins doing this, not your average moderator
 				check { hasPermission(Permission.Administrator) }
 
 				action {
-					val actionLogId: String?
-					// Try to select the guild from the database, with either return NoSuchElementException,
-					//  or the guild id
-					val alreadySet = DatabaseHelper.selectInConfig(guild!!.id, DatabaseManager.Config.guildId)
-
-					// If the database is empty, there is nothing set for this guild, so we move ahead with
-					// adding config else we inform the config is already set
-					if (alreadySet.equals("NoSuchElementException")) {
-						newSuspendedTransaction {
-							DatabaseManager.Config.insertIgnore {
-								it[guildId] = guild!!.id.toString()
-								it[moderatorsPing] = arguments.moderatorPing.id.toString()
-								it[supportTeam] = arguments.supportTeam?.id.toString()
-								it[modActionLog] = arguments.modActionLog.id.toString()
-								it[messageLogs] = arguments.messageLogs.id.toString()
-								it[supportChannel] = arguments.supportChannel?.id.toString()
-								it[joinChannel] = arguments.joinChannel.id.toString()
-							}
-						}
-
-						// Once inserted to the database we get the mod action log, so we can print a message to it
-						actionLogId = DatabaseHelper.selectInConfig(guild!!.id, DatabaseManager.Config.modActionLog)
+					// If an action log ID doesn't exist, set the config
+					// Otherwise, inform the user their config is already set
+
+					if (DatabaseHelper.selectInConfig(guild!!.id, "modActionLog") == null) {
+						val newConfig = ConfigData(
+							guild!!.id,
+							arguments.moderatorPing.id,
+							arguments.modActionLog.id,
+							arguments.messageLogs.id,
+							arguments.joinChannel.id,
+							arguments.supportChannel?.id,
+							arguments.supportTeam?.id,
+						)
+
+						DatabaseHelper.putInConfig(newConfig)
 
 						respond { content = "Config Set for Guild ID: ${guild!!.id}!" }
 
-						val actionLogChannel = guild?.getChannel(Snowflake(actionLogId!!)) as GuildMessageChannelBehavior
-						ResponseHelper.responseEmbedInChannel(
-							actionLogChannel,
-							"Configuration set!",
-							"An administrator has set a config for this guild!",
-							null,
-							user.asUser()
-						)
 					} else {
-						respond { content = "Your configuration is already set, clear it first before updating!" }
+						respond { content = "**Error:** There is already a configuration set for this guild!" }
+						return@action
 					}
+
+					// Log the config being set in the action log
+					val actionLogChannel = guild?.getChannel(arguments.modActionLog.id) as GuildMessageChannelBehavior
+					ResponseHelper.responseEmbedInChannel(
+						actionLogChannel,
+						"Configuration set!",
+						"An administrator has set a config for this guild!",
+						null,
+						user.asUser()
+					)
```

Comment:
Would it be possible to make this more verbose? As in, tell what config was set.